tao/tao_bridge/ral/BUILD

```diff
@@ -0,0 +1,82 @@
+package(default_visibility = ["//visibility:public"])
+load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
+load("@local_config_tf//:build_defs.bzl", "tf_copts")
+load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda", "cuda_library")
+
+cc_proto_library(
+    name = "compile_metadata",
+    srcs = [
+        "tensorflow/compiler/mlir/xla/compile_metadata.proto",
+    ],
+)
+
+filegroup(
+    name = "ral_headers",
+    srcs = glob(["tensorflow/**/*.h"]),
+)
+
+cc_library(
+    name = "bridge_ral",
+    srcs = [
+        "tensorflow/compiler/mlir/xla/compile_metadata.pb.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/common_context_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/stream_executor_based_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/tensorflow/tf_context_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/tensorflow/tf_kernel_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/device/cpu/cpu_driver.cc",
```

Comment:
srcs are in `tensorflow` directory. I know why, but it's strange.

Reply:
These are linked dirs... Originally they have BUILD file when build for tao_compiler_main in tf_community. When build for tao bridge, we already a tf headers and sos from installed tensorflow wheel, it will conflict with tf_community's headers with the same name, so I have to write a new BUILD file like this.